src/styles/calendars.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  margin: auto;
+  display: flex;
+  padding: #{$padding};
+  flex-wrap: wrap;
+  align-items: center;
+  width: #{$width--main};
+  gap: #{$gap--value};
+
+  &__cell {
+    background-color: #eee;
+    box-sizing: border-box;
+    height: #{$height--box};
+    width: #{$width--box};
+    border: 1px solid #000;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+      transition-duration: 0.5s;
```

Comment:
Move transition property to element, not only for `hover` effect, now the smoothness of the element works only when hovering over it, and when the cursor is removed, everything breaks off abruptly